### DISCUSSION TITLE
Nextjs 13 app directory

### DISCUSSION THREAD

**Discussion by aimarsouris** [keystonejs/keystone#8823] [Questions]
Hi, 
I just bumped into keystone and it seems to match many of my requirements, looks exciting! As the examples use the page directory, i was wondering if 'keystoneContext' also work with the new app directory? Thks

**Comment by aimarsouris** [comment]
got it: examples/framework-nextjs-app-directory/src/app/page.tsx

**Reply by borisno2** [reply]
Hi @aimarsouris you might also find my project https://github.com/borisno2/on-the-hill-drama-club interesting too. I have also recorded a few youtube videos re deploying to Vercel
